app/views/polls/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with model: poll do |form| %>
+<%= form_for @poll do |form| %>
```

Comment:
`form_for` is the old way of creating forms. You should be able to revert this to `form_with` and is should carry on working. I suspect you made this change when you were trying to figure out why everything wasn't working.

`form_with` is the replacement for `form_for` and `form_tag` and I think it is part of either Rails 6 or Rails 7